src/Writer/MySQL.php

```diff
@@ -171,16 +173,13 @@ protected function emptyToNullOrDefault(array $table): string
         $expressions = array_map(function ($column) {
             switch (strtolower($column['type'])) {
                 case 'date':
-                    $emptyValue = '0000-00-00';
-                    $defaultValue = !empty($column['default']) ? $this->db->quote($column['default']) : 'NULL';
-                    break;
                 case 'datetime':
-                    $emptyValue = '0000-00-00 00:00:00';
+                    $emptyValue = '';
```

Comment:
`$emptyValue` je v oboch vetvach `case` rovnake, ... dal by som to rovno do `sprintf`, netreba ani do premennej.

Reply:
jop, pravda